src/docs/product/discover-queries/uncover-trends.mdx

```diff
@@ -0,0 +1,118 @@
+
+4. Edit the "Results" table columns to show the number of events [`count()`], unique issues [`count_unique(issue)`], and impacted users [`count_unique(user)`] per project.
+
+   ![Issues per project](001.png)
+
+5. With these columns set, you might want to look for the project with the most issues. Click on the `COUNT_UNIQUE(ISSUE)` column header to order the line items accordingly.
+
+6. Hover your cursor over the top project line item, click on the actions icon that displays, and select "Add to filter". This will narrow your results even more so that you can continue looking into the issues of that specific project.
+
+   ![Project with most issues](002.png)
+
+7. Now let's see what those issues are. Change the table columns to display:
+
+   - `issue` (the issue id)
+   - `title` (issue title)
+   - `count()` (number of events)
+   - `count_unique(user)` (number of unique users impacted by the issue)
+
+8. On each line item/issue, you can click on the issue name to open the **Issue Details** page and triage the issue. You can also click on the Open Group icon to continue querying through the events stack in the context of a specific issue.
+
+   ![Actions per line item](003.png)
+
+## Discover Through Errors
+
+The true power of Discover lies in its ability to query through all your error metadata, so you're no longer restricted to the context of issues or projects, and can reveal trends and insights across all your applications. Let's look at some examples.
+
+### Errors by URLs
+
+Create a report looking into all the errors occurring in the URL endpoints across your applications.
+
+- Search condition: `event.type:error has:url`
+- Table columns: `url`, `platform.name`, `count()`
+
+![Errors by URL](005.png)
+
+- Look into any one of the URL endpoints by clicking on "Add to filter" and changing the table columns.
+- Alternatively, filter URL patterns using wild cards:
+
+![Wildcard filter](009.png)
+
+### Unhandled Errors
+
+Whether your code is running on mobile, in browser, or on a server, an unhandled fatal error might crash your application. To find out where those crashes are happening, run the following query in Discover:
+
+- Search condition: `event.type:error handled:no level:fatal`
+- Table columns: `mechanism`, `platform.name`, `count()`
+
+  ![Crashes](006.png)
+
+To look deeper into one of the crash types:
+
+- Select one of the line item values and add it to the filter.
+- Modify the table columns to show the crash `message` and `count()`.
+
+  ![Native Crashes](007.png)
+
+### Files With Most Errors
+
+To find out which files in your codebase are generating the most errors, run the following query in Discover:
+
+- Search condition: `event.type:error has:stack.filename`
+- Table columns: `stack.filename`, `count()`
+
+![By File Names](010.png)
+
+You can continue exploring a specific filename by adding it to the filter and changing the table columns to display the major error culprits in that file:
+
+![Culprits](008.png)
+
```

Comment:
@wmak following our convo yesterday, can you review the Errors per Release section and let me know if it's accurate?